scripts/engineering/aggregate_phase44d_carry_aware_experimental_3x3_scaling.py

```
@@ -0,0 +1,203 @@
+                    row["primitive"],
+                    row["backend_variant"],
+                    str(row["steps"]),
+                    row["relation"],
+                    str(row["serialized_bytes"]),
+                    format_milliseconds(float(row["emit_ms"])),
+                    format_milliseconds(float(row["verify_ms"])),
+                    str(row["verified"]),
+                    str(row["note"]).replace("\t", " "),
+                ]
+            )
+        )
```

Ruff (0.15.11)

[warning] Use `list.extend` to create a transformed list

(PERF401)